INSTALL.md

```diff
@@ -1,24 +1,31 @@
 ## Installation
 
 ### Requirements
+
 - Linux, CUDA >= 11.7, Python >= 3.8, PyTorch >= 2.0.0 (our setup below is based on CUDA 11.8, Python 3.10, PyTorch 2.0.1; more recent versions should work too, but no guarantees)
 - Conda (anaconda / miniconda work well)
+- visual studio visual studio (2022/2019/2017) build tools
```

Comment:
Perhaps this was required for Windows only? (nit: `visual studio` is repeated twice)